tests/acceptance/lazy-routeable-engine-test.js

```diff
@@ -14,28 +16,23 @@ moduleForAcceptance('Acceptance | lazy routable engine', {
 
     // We stub out the loader methods so that we can verify what they're doing.
     const module = this;
+    this.loader = this.application.__container__.lookup('service:asset-loader');
```

Comment:
In general it is better to use `this.application.__deprecatedInstance__` instead (as represents an object that is already public API).

Reply:
Nice! didn't know that